CorsixTH/Lua/ui.lua

```diff
@@ -235,9 +235,13 @@ function UI:setupGlobalKeyHandlers()
   self:addKeyHandler({"alt", "f4"}, self, self.exitApplication)
   self:addKeyHandler({"shift", "f10"}, self, self.resetApp)
 
-  if self.app.config.debug then
-    self:addKeyHandler("f12", self, self.showLuaConsole)
-    self:addKeyHandler({"shift", "d"}, self, self.runDebugScript)
+  self:addOrRemoveDebugModeKeyHandlers()
+end
+
+function UI:connectDebugger()
+  local error_message = TheApp:connectDebugger()
+  if error_message then
+    self.ui:addWindow(UIInformation(self.ui, {error_message}))
```

Comment:
@sadger Thanks for posting your fix for this mistake, I will submit a pull request for it.